packages/consensus/src/types/transaction.cairo

```diff
@@ -52,9 +52,6 @@ pub struct TxIn {
 
 /// A reference to an unspent transaction output (UTXO).
 ///
-/// NOTE that `data` and `block_height` meta fields are not serialized with the rest of
```

Comment:
This is about https://github.com/keep-starknet-strange/raito/blob/1d2a115d872ea3b6e68eab345b4dde4d50f20d1a/packages/consensus/src/codec.cairo#L71

Reply:
oh.